requirements/lint.in

```diff
@@ -3,4 +3,4 @@ flake8
 html-linter
 pep8-naming
 black==19.10b0
-isort
+isort>=5
```

Comment:
This is not needed in the strictest sense, but IF we ever had another tool that required isort <5, at least we'd get a loud error. lint.txt is already good (https://github.com/pypa/warehouse/pull/8288)